src/pages/generalSettings/GeneralSettings.jsx

```diff
@@ -238,12 +320,16 @@ export default function GeneralSettings() {
                   };
                 }
               });
-              putSiteSettings(currentValues);
-              if (logoPostData) postSettingsAsset(logoPostData);
+              putSiteSettings({ data: currentValues });
+              if (logoPostData)
+                postSettingsAsset({
+                  data: logoPostData,
+                });
             }}
             style={{ marginTop: 12 }}
             display="primary"
             loading={loading}
+            disabled={!allValid}
```

Comment:
is `allValid` a good name? seems like this variable is more about whether intelligentAgentFields are valid

Reply:
Done.